scripts/travisPublishDocs.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/sh
+
+# Set identity
+git config --global user.email "travis@travis-ci.org"
+git config --global user.name "Travis"
+
+# Publish docs
+mkdir ../gh-pages
+cp -r doc/_build/html/* ../gh-pages/
+cd ../gh-pages
+
+# Add branch
+git init
+git remote add origin https://${GH_TOKEN}@github.com/hyperwallet/python-sdk.git > /dev/null
```

Comment:
Use `CI_USER_TOKEN` instead of `GH_TOKEN`

Reply:
Update to hyperwallet-ios-ui-sdk.git instead of python-sdk.git

Reply:
updated